### PR TITLE
Function delete and edit prototype

### PR DESCRIPTION
**Pull request by tomoki0120**
# WHAT

create_a_delete_function_and_an_edit_function
# WHY

For_edit_and_delete_prototype
# DEADLINE

9/21/15:00

### CODE REVIEW

**Comment by tomoki0120** [comment]
https://gyazo.com/908a811effc27406f0e6f66eb9048152

**Comment by tomoki0120** [comment]
https://gyazo.com/f12d05afc7351eb0061f102fa534e151

**Comment by tomoki0120** [comment]
https://gyazo.com/7c116ba57d57b3efabade32836897b7b

**Comment by tomoki0120** [comment]
https://gyazo.com/cb7df3d19ce7ac2647d7dbb6527274e0

**Comment by tomoki0120** [comment]
https://gyazo.com/7b0d80575d279a858b4b2e5465e24f84

File: app/controllers/prototypes_controller.rb [hunk 1]
```diff
@@ -1,5 +1,8 @@
 class PrototypesController < ApplicationController
 
+  before_action :set_prototype, only: [:edit,:update,:destroy,:show]
+  before_action :set_photo, only: [:show,:edit]
```
Comment (takuma3110) [inline app/controllers/prototypes_controller.rb:4]: good!

File: app/models/proto.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 class Proto < ActiveRecord::Base
-  has_many :images
+  has_many :images, dependent: :delete_all
```
Comment (takuma3110) [inline app/models/proto.rb:2]: ここ、
dependent: :deleteじゃなくて、
dependent: :delete_allにしてるのはなんでかな？

**Pull request merged**